distributions/otelcol-k8s/manifest.yaml

```diff
@@ -0,0 +1,71 @@
+dist:
+  module: github.com/open-telemetry/opentelemetry-collector-releases/k8s
+  name: otelcol-k8s
+  description: OpenTelemetry Collector for Kubernetes
+  version: 0.96.0
+  output_path: ./_build
+  otelcol_version: 0.96.0
+
+extensions:
+  - gomod: go.opentelemetry.io/collector/extension/zpagesextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/basicauthextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/bearertokenauthextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/headerssetterextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/healthcheckextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/httpforwarderextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/oauth2clientauthextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/hostobserver v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/k8sobserver v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/oidcauthextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/pprofextension v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage v0.96.0
+    import: github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage/filestorage
+
+exporters:
+  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.96.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.96.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/loadbalancingexporter v0.96.0
```

Comment:
Should we include https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/fileexporter ? This can be helpful for debugging.

Reply:
I think this component would be ok.  I can update the criteria to state `Only exporters that use OTLP are allowed unless the exporter is useful for troubleshooting`.